tests/07_configadapter_firefox.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# vi:ts=2 sw=2 sts=2
+
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the licence, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Authors: Alberto Ruiz <aruiz@redhat.com>
+#          Oliver Gutiérrez <ogutierrez@redhat.com>
+
+import os
+import sys
+import tempfile
+import shutil
+import json
+import unittest
+
+import gi
+from gi.repository import GLib
+
+sys.path.append(os.path.join(os.environ['TOPSRCDIR'], 'src'))
+
+import fleetcommanderclient.configadapters.firefox
+from fleetcommanderclient.configadapters.firefox import FirefoxConfigAdapter
+
+PREFS_FILE_CONTENTS = """pref("accessibility.typeaheadfind.flashBar", 0);
+pref("beacon.enabled", false);
```

Comment:
the content in .mozilla/firefox/***.profile/prefs.js uses user_pref rather than pref, we should probably test for that instead

Reply:
Nevermind this is the client. Forget what I just said.